python/tvm/micro/testing/pytest_plugin.py

```diff
@@ -26,14 +26,18 @@
 
 from tvm.contrib.utils import tempdir
 
-from .utils import get_supported_boards
+from .utils import get_supported_platforms, get_supported_boards
 
 
 def pytest_addoption(parser):
     """Adds more pytest arguments"""
+    parser.addoption(
```

Comment:
doesn't this force every test to have --platform as an argument? In that case we don't want this for tests that are specific to Zephyr/Arduino

Reply:
Nope! Only tests that include the `platform` fixture will require the `--platform` argument, and likewise for `board`.

Reply:
cool, thanks for clarification!